### PR TITLE
changed comment line

### PR DESCRIPTION
**Pull request by sweavs111**

### CODE REVIEW

**Comment by sweavs111** [comment]
Directory changed to "bedpeOverlap" and all instances of 'bedPe' have been changed to 'bedpe'. There are still instances of BedPe because that is how the struct is named in the bedpe.go file. I can go about changing all of those if we would like. But from outward appearance the issue is fixed

File: cmd/bedpeOverlap/bedpeOverlap_test.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ func TestBedPeOverlap(t *testing.T) {
 	for _, v := range BedPeOverlapTests {
 		bedpeOverlap(v.selectFile, v.inBedPe, v.outBedPe, v.bedSelect, v.overlapThreshold, v.overlapBoth)
 		if !fileio.AreEqual(v.outBedPe, v.expectedFile) {
-			t.Errorf("Error in bedPeOverlap.")
+			t.Errorf("Error in bedpeOverlap.")
```
Comment (edotau) [inline cmd/bedpeOverlap/bedpeOverlap_test.go:31]: A more descriptive error message can help others debug in the future. Perhaps something like
```suggestion
			t.Errorf("Error: bedpeOverlap files %s and %s are not equal to one another...",  v.outBedPe, v.expectedFile)
```

**Review by edotau — DISMISSED** [review]
lgtm!

**Comment by rimangan** [comment]
Looks good, I committed Eric's suggestion. It looks like we're still failing tests related to files not found

**Comment by edotau** [comment]
> Looks good, I committed Eric's suggestion. It looks like we're still failing tests related to files not found

I made a fork of this PR and fixed the failed tests in this PR: #794

**Review by rimangan — APPROVED** [review]

**Review by RavenYL — APPROVED** [review]

**Pull request merged**